eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Rules/Utxos.hs

```diff
@@ -281,7 +281,7 @@ scriptsNotValidateTransition = do
   pure
     $! us
       { _utxo = UTxO utxoKeep,
-        _fees = fees <> Val.coin (balance (UTxO utxoDel)),
+        _fees = fees <> coinBalance (UTxO utxoDel),
```

Comment:
I love this new, clearer language.

Reply:
It is not only clearer, but it is also 100x faster :wink:

Reply:
:metal: